ansible/inventory.sample

```diff
@@ -15,12 +15,17 @@ drive
 ansible-worker
 jenkins-server
 
+[gerrit:children]
+zuul
```

Comment:
`zuul` shouldn't be a part of `gerrit` group, because if `zuul` host would be in inventory then `post-recheck-comment` playbook would be run on gerrit server and zuul server as well.

We propose to define two groups:
- `gerrit` - only varialbes
- `gerrit-servers` - for gerrit hosts

Then we could inherit variables like this:

```
[gerrit:children]
gerrit-server
zuul
```

Reply:
Agreed. Although using `gerrit` only for variables seems weird. Perhaps we can use the following hierarchy?

* `gerrit_variables`
    * `zuul`
    * `gerrit` (or `gerrit-server`)

Reply:
It might be okay. Other proposal is `gerrit-clients`. This way name of the group represents hosts' purpose.

Reply:
Since now we have gerrit in the inventory, we can remove the `gerrit-variables` and make `zuul` pull all the needed variables from `gerrit`'s vars. But it's out of scope if this PR, I guess.